contracts/StorageManager.sol

```diff
@@ -16,27 +16,27 @@ contract StorageManager {
     /*
     Offer represents:
      - utilizedCapacity: how much is capacity is utilized in Offer.
-     - totalCapacity: total amount of bytes offered.
+     - totalCapacity: total amount of mega-bytes (MB) offered.
      - billingPlans: maps a billing period to a billing price. When a price is 0, the period is not offered.
      - agreementRegistry: the proposed and accepted Agreement
     */
     struct Offer {
         uint128 utilizedCapacity;
         uint128 totalCapacity;
-        mapping(uint64 => uint64) billingPlans;
+        mapping(uint64 => uint128) billingPlans;
         mapping(bytes32 => Agreement) agreementRegistry; // link to agreement that are accepted under this offer
     }
 
     /*
     Agreement represents:
      - billingPrice: price per byte that is collected per each period.
      - billingPeriod: period how often billing happens.
-     - size: allocated size for the Agreement (in bytes, rounded up)
+     - size: allocated size for the Agreement (in MB, rounded up)
      - availableFunds: funds available for the billing of the Agreement.
      - lastPayoutDate: When was the last time Provider was payed out. Zero either means non-existing or terminated Agreement.
     */
     struct Agreement {
-        uint64 billingPrice;
+        uint128 billingPrice;
         uint64 billingPeriod;
         uint256 availableFunds;
         uint128 size;
```

Comment:
You may also make this a `uint64`, not making the struct bigger.

Reply:
This is great feedback! Thanks, I will downsize the `size`